skins/cat17/templates/contact_form.html.twig

```diff
@@ -26,6 +26,18 @@
 							<label for="email" class="sr-only">{$ 'contact_email'|trans $}</label>
 							{$ util.form_error('email', errors) $}
 						</div>
+						<select class="no-outline personal-title" id="title" name="title"
```

Comment:
Same here, the `<select>` contents can reduced to fewer lines using for statement.
https://twig.symfony.com/doc/2.x/tags/for.html